domains/squid.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "funnysquib",
+        "email": "mac.chips89@gmail.com"
+    },
+    "records": {
+        "CNAME": "https://funnysquib.github.io/website"
```

CNAME doesnt allow/accept trailing slashes (/)
```suggestion
        "CNAME": "funnysquib.github.io"
```